flocker/node/testtools.py

```diff
@@ -52,6 +55,42 @@ def docker_accessible():
     ))
 
 
+def require_docker_version(minimum_docker_version, message='N/A'):
```

Comment:
:question: Might as well make `message` compulsory for now, since it's provided in the only use.